crates/vm/vm.rs

```diff
@@ -169,7 +169,7 @@ cfg_if::cfg_if! {
 
             let env = Environment {
                 origin: tx.sender(),
-                consumed_gas: U256::from(21000), // Base gas cost for a transaction
+                consumed_gas: U256::zero(), // Base gas cost for a transaction
```

Comment:
Why this?

Reply:
Because TX_BASE_COST is added in the intrinsic gas function. I have to remove that comment though